packages/application-generic/src/usecases/add-job/add-job.usecase.ts

```diff
@@ -30,7 +29,8 @@ const LOG_CONTEXT = 'AddJob';
 export class AddJob {
   constructor(
     private jobRepository: JobRepository,
-    private queueService: QueueService,
+    @Inject(forwardRef(() => StandardQueueService))
```

Comment:
AddJob is part of a severe circular dependency. We need to find a strategy to solve it. Might involve redo the logic flows.